hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -1303,6 +1303,8 @@
       hwaccess: "hro",
       hwqe: "true",
       regwen: "REGWEN",
+      tags: [// Internal HW can modify status register
```

Comment:
dito

Reply:
updated comment to reflect not wanted a fatal alert.